geoportailv3/static/js/maincontroller.js

```diff
@@ -536,6 +544,8 @@ app.MainController = function(
         this['layersOpen'] = true;
       }
       this.showTab('a[href=\'#mylayers\']');
+    } else {
+      this.appMymapsOffline_.checkDataFormat();
```

Comment:
This could be done in the mymapsserviceoffline constructor.
We don't really care if the user is offline or not.